src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss

```diff
@@ -244,3 +244,26 @@ div.onboarding-product-step {
 		}
 	}
 }
+
+.nfd-onboarding-experience-step {
```

Comment:
![Screenshot 2023-07-26 at 6 16 07 PM](https://github.com/newfold-labs/wp-module-onboarding/assets/38878906/6ee6e230-ec94-4fc6-be76-3c0e80f598d3)
This CSS selector is modifying the experience step as well, @ramyakrishnai can you please check and fix this. We should be able to use `.radio-control-tax-step`